.circleci/config.yml

```diff
@@ -136,6 +139,20 @@ commands:
       - setup_remote_docker
       - setup_env
       - checkout
+      - when:
+          condition: <<parameters.rebase_with_master>>
```

Comment:
rebasing with master based on condition (false by default, but true when built for all branches other than master and release)